english/untrustworthy/untrustworthy.md

```diff
@@ -0,0 +1,2 @@
+"If you want something done right, you have to do it yourself."
+Don't trust other people to do important things for you. You have to do things yourself to control the quality of the results.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.